pandas/core/dtypes/common.py

```diff
@@ -304,34 +304,6 @@ def is_offsetlike(arr_or_obj):
     return False
 
 
-def is_period(arr):
```

Comment:
is this not generally exposed to the user / higher level? I don't remember why we added this (and it has the wrong name)